src/scripts.js

```diff
@@ -16,9 +19,27 @@ const favoriteRecipesSection = document.getElementById('favoriteRecipesSection')
 
 const weeklyRecipesSection = document.getElementById('weeklyRecipesSection');
 
+let searchBar = document.querySelector('.favorites-view__favorites');
+let mainGlide = document.querySelector('.search-ingredient-glide__images-container');
```

Comment:
These should also be `const` unless they're reassigned somewhere. From what I can see, it doesn't look like they are.